### ISSUE TITLE
Consider removing the replace directive in `go.mod`

### ISSUE THREAD

**Issue by ghost** [wakatime/wakatime-cli#1109]
Hi. I tried to install wakatime-cli by [go install].

```console
go install github.com/wakatime/wakatime-cli@v1.102.1
```

```
go: github.com/wakatime/wakatime-cli@v1.102.1 (in github.com/wakatime/wakatime-cli@v1.102.1):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

Please consider removing the replace directive. Changes in [wakatime/goInfo] already been merged to [matishsiao/goInfo].

https://github.com/wakatime/wakatime-cli/blob/660b2b9702f3bc598e96cb8b8e9f459a6c2928c3/go.mod#L58

Thanks for all amazing work in wakatime-cli and goInfo.

[wakatime/goInfo]: https://github.com/wakatime/goInfo
[matishsiao/goInfo]: https://github.com/matishsiao/goInfo

[go install]: https://go.dev/ref/mod#go-install

**Comment by alanhamlett** [comment]
Done in #1110

**Comment by eli-yip** [comment]
The same problem happens again: https://github.com/wakatime/wakatime-cli/blob/65ef62fb3a900cefd617270bf539b13880596978/go.mod#L33

**Comment by gandarez** [comment]
Why do you need to import wakatime-cli?

**Comment by eli-yip** [comment]
> Why do you need to import wakatime-cli?

It's not about import but install.

When you run `go install github.com/wakatime/wakatime-cli@latest`,  some errors will be throwed because `replace` statement in go.mod.

```
# go install github.com/wakatime/wakatime-cli@latest
go: github.com/wakatime/wakatime-cli@latest (in github.com/wakatime/wakatime-cli@v1.115.5):
        The go.mod file for the module providing named packages contains one or
        more replace directives. It must not contain directives that would cause
        it to be interpreted differently than if it were the main module.
```

**Comment by gandarez** [comment]
@alanhamlett why bbolt is forked?

**Comment by alanhamlett** [comment]
> [@alanhamlett](https://github.com/alanhamlett) why bbolt is forked?

We needed a bugfix not yet released: https://github.com/wakatime/wakatime-cli/pull/1178

**Comment by alanhamlett** [comment]
It was released last week so we can remove the replace and upgrade bolt.

**Comment by alanhamlett** [comment]
Fixed with #1186.
